### ISSUE TITLE
Detect AAA non-fuctional

### ISSUE THREAD

**Issue by psouza4** [psouza4/mediacentermaster#1669]
Originally reported on Google Code with ID 1677
```
What steps will reproduce the issue?
1.Enable absolute order detection
2.Ensure anime has at least 3 digits (i.e. episode 1 becomes 001)
3.Process downloads

4. Disable absolute order, repeat steps 2/3 with same results


What is the expected output?  
Correctly parsed file based on absolute order

What do you see instead?
Error message

What is the exact version of the Media Center Master that you are using?
2.14.13415.810


Please provide any additional information below:

MCM:  Now working on [HorribleSubs] Fate Stay Night - Unlimited Blade Works - 001 [1080p].mkv...
MCM:  Automatically detecting [HorribleSubs] Fate Stay Night - Unlimited Blade Works
- 001 [1080p].mkv...
MCM:      [HorribleSubs] Fate Stay Night - Unlimited Blade Works - 001 [1080p].mkv
MCM:  ... this file was not detectable
MCM:  I'm not sure if this file is a T.V. episode or a movie -- I can't detect it as
either:  [HorribleSubs] Fate Stay Night - Unlimited Blade Works - 001 [1080p].mkv
MCM:  The file is not named in a way I understand.  If this is a T.V. episode and you
have a scan folder set up for it, then you may want to add a T.V. detection override
settings to manually map a series description to its ID.
-----------------------------------------------------------------------------------------
MCM:  Now working on [HorribleSubs] Fate Stay Night - Unlimited Blade Works - 002 [1080p].mkv...
MCM:  Automatically detecting [HorribleSubs] Fate Stay Night - Unlimited Blade Works
- 002 [1080p].mkv...
MCM:      [HorribleSubs] Fate Stay Night - Unlimited Blade Works - 002 [1080p].mkv
MCM:  ... this file was not detectable
MCM:  I'm not sure if this file is a T.V. episode or a movie -- I can't detect it as
either:  [HorribleSubs] Fate Stay Night - Unlimited Blade Works - 002 [1080p].mkv
MCM:  The file is not named in a way I understand.  If this is a T.V. episode and you
have a scan folder set up for it, then you may want to add a T.V. detection override
settings to manually map a series description to its ID.
-----------------------------------------------------------------------------------------
MCM:  Now working on [HorribleSubs] Seraph of the End 011 [1080p].mkv...
MCM:  Automatically detecting [HorribleSubs] Seraph of the End 011 [1080p].mkv...
MCM:      [HorribleSubs] Seraph of the End 011 [1080p].mkv
MCM:  ... this file was not detectable
MCM:  I'm not sure if this file is a T.V. episode or a movie -- I can't detect it as
either:  [HorribleSubs] Seraph of the End 011 [1080p].mkv
MCM:  The file is not named in a way I understand.  If this is a T.V. episode and you
have a scan folder set up for it, then you may want to add a T.V. detection override
settings to manually map a series description to its ID.
-----------------------------------------------------------------------------------------


Additionally, if the numbering is changed to SEE numbering such as 101 instead of 001
while AAA dectection is enabled or disabled.


MCM:  Now working on [HorribleSubs] Fate Stay Night - Unlimited Blade Works - 101 [1080p].mkv...
thetvdb.com  Searching by season 1, episode 1
MCM:  ... this is a T.V. show episode (detected using quick method)
MCM:  Moving and renaming to:
MCM:      Fate-Stay Night, Unlimited Blade Works\Season 1
MCM:      Fate-Stay Night, Unlimited Blade Works - 1x01 - A Winter's Day, A Fateful
Night.mkv
MCM:  Fetching metadata for the new episode...
-----------------------------------------------------------------------------------------
MCM:  [F#-1] now working on Fate-Stay Night, Unlimited Blade Works
thetvdb.com  Updated base metadata for this show
thetvdb.com  Fetching season 1 poster...
thetvdb.com  Fetching season 1 banner...
thetvdb.com  Fetching data for season 1, episode 1
thetvdb.com  "A Winter's Day, A Fateful Night"
thetvdb.com  Fetching season 1, episode 1 thumbnail...
MCM:  metadata
MCM:  ... media details saved using VidInfo utility


Issue with Debug

MCM:  Now working on [HorribleSubs] Fate Stay Night - Unlimited Blade Works - 004 [1080p].mkv...
MCM:  Automatically detecting [HorribleSubs] Fate Stay Night - Unlimited Blade Works
- 004 [1080p].mkv...
MCM:      [HorribleSubs] Fate Stay Night - Unlimited Blade Works - 004 [1080p].mkv
[16:56:05.296] DEBUG:  Getting video details...
[16:56:05.336] DEBUG:  Saved to cache: D:\
[16:56:05.463] DEBUG:  ... Video quality: Excellent (1080p HD)
[16:56:05.463] DEBUG:  ... Video dimensions: 1920x1080 (1.778:1 / 1.78:1)
[16:56:05.463] DEBUG:  ... Video bitrate: 3191
[16:56:05.463] DEBUG:  ... Video framerate: 23.976
[16:56:05.463] DEBUG:  ... Video codec: H.264 (AVC)
[16:56:05.464] DEBUG:  ... Video duration: 24 minutes
[16:56:05.464] DEBUG:  ... Video scan type: Progressive
[16:56:05.465] DEBUG:  ... Audio stream language: 
[16:56:05.465] DEBUG:  ... Audio stream channels: 2
[16:56:05.465] DEBUG:  ... Audio stream bitrate: 0
[16:56:05.466] DEBUG:  ... Audio stream frequency: 44100
[16:56:05.467] DEBUG:  ... Audio stream codec: AAC (AAC)
[16:56:05.468] DEBUG:  Operations ... video details inspection took 0.2 seconds
[16:56:05.646] DEBUG:  Saved to cache: D:\
[16:56:06.120] DEBUG:  Series ID: 278626
[16:56:06.120] DEBUG:  Series name: Fate Stay Night Unlimited Blade Works
[16:56:06.120] DEBUG:  Season: 
[16:56:06.120] DEBUG:  Episode: 
[16:56:06.120] DEBUG:  Episode 2: 
[16:56:06.120] DEBUG:  Air date: 
[16:56:06.382] DEBUG:  Looking for override for Fate Stay Night Unlimited Blade Works...
[16:56:06.382] DEBUG:  ... none found (will search online instead)
[16:56:06.500] Fetching URL (GET):  http://thetvdb.com/api/767CE15749DAFFA2/series/278626/default/0/0/en.xml
MCM:  ... this file was not detectable
[16:56:06.533] DEBUG:  Detected:		False
[16:56:06.533] DEBUG:  ... type:		Unknown
[16:56:06.533] DEBUG:  ... movie?		False
[16:56:06.533] DEBUG:  ... episode?		False
[16:56:06.533] DEBUG:  ... episode stub?	False
MCM:  I'm not sure if this file is a T.V. episode or a movie -- I can't detect it as
either:  [HorribleSubs] Fate Stay Night - Unlimited Blade Works - 004 [1080p].mkv
MCM:  The file is not named in a way I understand.  If this is a T.V. episode and you
have a scan folder set up for it, then you may want to add a T.V. detection override
settings to manually map a series description to its ID.
```

Reported by `Darkangeldbgt` on 2015-06-13 15:57:25

**Comment by robleedes** [comment]
If the "Detect ### pattern in filenames as AAA instead of shorthand SEE" is an obsolete setting due to the addition of the series level setting and it is no longer working, I believe it should be removed from the UI.

**Comment by psouza4** [comment]
Fixed and released in version 2.14.25615.885.
